app/public/scripts/utils.js

```diff
@@ -34,7 +35,7 @@ function myInitCode() {
   if(deleteArticleButton) deleteArticleButton.addEventListener('click', deleteHandler)
   if (hamburger) hamburger.addEventListener('click', openHamburgerMenu)
   if(uploadArticleImageButton) uploadArticleImageButton.addEventListener('click', uploadImage)
-  
+  if(unsubscribeButton) unsubscribeButton.addEventListener('click', () => clickHandler('/unsubscribe'))
```

Comment:
Use `<a href="..." >`

Reply:
`clickHandler` can be removed in general, as you should use `a href` whenever possible in for reidrections with no additional logic